src/ensembl/src/content/app/entity-viewer/shared/helpers/transcripts-sorter.ts

```diff
@@ -27,11 +28,13 @@ import {
 
 import { SortingRule } from 'src/content/app/entity-viewer/state/gene-view/transcripts/geneViewTranscriptsSlice';
 
-import { SliceWithLocationOnly } from 'src/shared/types/thoas/slice';
+import { Slice } from 'src/shared/types/thoas/slice';
+
+type SliceWithOnlyLength = Pick2<Slice, 'location', 'length'>;
```

Comment:
Would it be worth defining subset types like these within the `slice.ts` file?

Reply:
Not sure. It depends on how commonly such types are reused or how much they make sense on their own. I didn't feel a strong urge to define such subtypes in the thoas-related types files.